common/shlibs

```diff
@@ -3440,8 +3440,8 @@ libmedC.so.11 libmed-4.0.0_1
 libmed.so.11 libmed-4.0.0_1
 libeog.so eog-3.28.0_4
 libfreexl.so.1 freexl-1.0.5_1
-libgeos-3.9.0.so geos-3.9.0_1
-libgeos_c.so.1 geos-3.9.0_1
+libgeos.so.3.11.1 geos-3.11.1_1
```

Comment:
because nothing uses this shlib, we can remove this line. This also means we don't need to rebuild anything and the change to the next line (libgeos_c...) can be reverted

Reply:
Don't worry, thanks. How about now?